examples/ts/proxy/README.md

```diff
@@ -0,0 +1,8 @@
+This directory contains a very simple proxy setup to allow a front-end to use BitGo SDK methods with a non-BitGo back-end.
```

Comment:
This should point out the difference between why you would choose this option vs. using the already built out express app in the modules folder.